src/config.js

```diff
@@ -98,6 +98,16 @@ export default Object.assign(
      * The field being displayed on login screen.
      */
     BUILD_VERSION: 'dev: ' + packageJson.version,
+
+    /**
+     * Module scheme to use. The module scheme will be used to combine module
+     * set of the application.
+     *
+     * TODO: move out of the config and find a way to install the scheme as
+     * plugin. Also would be fine if pageModule will be provided without using
```

Comment:
tmp: Also would be fine.. part seems to be resolved, isn't it?

Reply:
removed the TODO